### PR TITLE
Subscriptions fixes

### PR DESCRIPTION
**Pull request by a-alle**
# Description

Fixes minor issues caught by running existing integration tests with subscriptions enabled:
- unwind over empty [] meta does not return any data
- `meta not defined` cypher error

## Complexity

Complexity: Low

# Checklist

The following requirements should have been met (depending on the changes in the branch):

- [ ] Documentation has been updated
- [ ] TCK tests have been updated
- [ ] Integration tests have been updated
- [ ] Example applications have been updated
- [ ] New files have copyright header
- [ ] CLA (https://neo4j.com/developer/cla/) has been signed

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Made some changes to the GraphQL Toolbox? Or made some changes to the core library perhaps?

Check out your changes at https://neo4j-graphql-toolbox-stage-2544.surge.sh

Use for instance a local dbms hosted on [Neo4j Desktop](https://neo4j.com/download/) or a cloud-hosted [Aura Free](https://neo4j.com/cloud/aura-free/) dbms.

File: packages/graphql/src/translate/create-connect-and-params.ts [hunk 1]
```diff
@@ -308,6 +308,10 @@ function createConnectAndParams({
             if (relValidationStrs.length) {
                 subquery.push(`\tWITH ${[...filterMetaVariable(withVars), nodeName].join(", ")}${innerMetaStr}`);
                 subquery.push(relValidationStrs.join("\n"));
+
+                if (context.subscriptionsEnabled) {
+                    innerMetaStr = ", meta";
+                }
```
Comment (angrykoala) [inline packages/graphql/src/translate/create-connect-and-params.ts:314]: Painful, hopefully we will be able to move this to cypher builder soon

File: packages/graphql/tests/integration/subscriptions/update/top-level-where.int.test.ts [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+
+        expect(result.errors).toBeFalsy();
+        expect(result.data).toEqual({
+            [postType.operations.update]: {
+                [postType.plural]: expect.toBeArrayOfSize(1),
+            },
+        });
+    });
+
+    test.only("Top-level OR", async () => {
```
Comment (angrykoala) [inline packages/graphql/tests/integration/subscriptions/update/top-level-where.int.test.ts:143]:
```suggestion
    test("Top-level OR", async () => {
```

**Review by angrykoala — DISMISSED** [review]
Looks good, just an eslint error to fix

**Review by darrellwarde — APPROVED** [review]
LGTM!

**Pull request merged**